addons/knowledge/static/src/js/widgets/knowledge_dialogs.js

```diff
@@ -0,0 +1,186 @@
+odoo.define('knowledge.dialogs', function (require) {
+'use strict';
+
+const Dialog = require('web.Dialog');
+const { _t } = require('web.core');
+
+const MoveArticleToDialog = Dialog.extend({
+    template: 'knowledge.knowledge_move_article_to_modal',
+    /**
+     * @override
+     * @param {Widget} parent
+     * @param {Object} options
+     * @param {Object} props
+     */
+    init: function (parent, options, props) {
+        // Set default options:
+        options.title = options.title || _t('Move an Article');
+        options.buttons = options.buttons || [{
+            text: _t('Ok'),
+            classes: 'btn-primary',
+            click: this.save.bind(this)
+        }, {
+            text: _t('Cancel'),
+            close: true
+        }];
+        this._super(...arguments);
+        this.props = props;
+        // Set template variables:
+        const { state } = props;
+        this.article_name = state.data.name;
+    },
+
+    /**
+     * @override
+     * @returns
+     */
+    start: async function () {
+        const result = await this._super(...arguments);
+        this.initSelect2();
+        return result;
+    },
+
+    /**
+     * @returns {JQuery}
+     */
+    getInput: function () {
+        return this.$el.find('input');
+    },
+
+    /**
+     * @returns {String}
+     */
+    getLoggedUserPicture: function () {
+        const { context } = this.props.state;
+        const origin = window.location.origin;
+        return `${origin}/web/image?model=res.users&field=avatar_128&id=${context.uid}`;
+    },
```

Comment:
You should be able to retrieve the uid using "sessuin.uid" (you can grep for examples).

Also, not sure you need to specify the origin, a relative URL should work.
This becomes something like: `/web/image?model=res.users&field=avatar_128&id=${session.uid}`

Which is a one-liner, meaning no need for a function.
Less functions = less code = happy reviewer !